api/envoy/config/overload/v3/overload.proto

```diff
@@ -50,10 +50,20 @@ message ThresholdTrigger {
       "envoy.config.overload.v2alpha.ThresholdTrigger";
 
   // If the resource pressure is greater than or equal to this value, the trigger
-  // will fire.
+  // will enter saturation.
   double value = 1 [(validate.rules).double = {lte: 1.0 gte: 0.0}];
 }
 
+message RangeTrigger {
+  // If the resource pressure is greater than this value, the trigger will be in the
+  // :ref:`scaling <arch_overview_overload_manager-triggers-state>` state with value
+  // `(pressure - min_value) / (max_value - min_value)`.
+  double min_value = 1 [(validate.rules).double = {lte: 1.0 gte: 0.0}];
```

Comment:
My personal preference would be to:
1) s/min_value/scaling_threshold
2) s/max_value/saturation_threshold

And possibly s/RangeTrigger/ScaledTrigger or something like this. WDYT?

I think what we implement here is not actually what most people would expect by just reading the names.

(Whatever we decide here needs a release note.)

Reply:
Yep, ScaledTrigger sounds good. Updated here and throughout.